tests/run/test_named_expressions.py

```diff
@@ -0,0 +1,485 @@
+# copied from cpython with minimal modifications (mainly exec->cython.inline, and a exception strings)
```

Comment:
Seems to need a `pure38` tag.

Reply:
I've added a `pure38` tag, but it seems to be ignored (possibly because it's not a doctest?)